app/mailers/report_mailer.rb

```diff
@@ -1,5 +1,5 @@
 class ReportMailer < ActionMailer::Base
-  default from: AppConfig.mail.sender_address
+  default from: "#{AppConfig.settings.pod_name} <#{AppConfig.mail.sender_address}>"
```

Comment:
Here are the quotes missing, should be `"\"#{AppConfig.settings.pod_name}\" <#{AppConfig.mail.sender_address}>"`.